Leanplum-SDK/Classes/Notifications/Push/LPPushNotificationsHandler.m

```diff
@@ -313,7 +313,15 @@ - (void)maybePerformNotificationActions:(NSDictionary *)userInfo
     };
 
     if (!active) {
-        handleNotificationBlock();
+        if ([Leanplum hasStarted]) {
+            handleNotificationBlock();
+        } else {
+            [Leanplum onStartResponse:^(BOOL success) {
+                LP_END_USER_CODE
+                handleNotificationBlock();
+                LP_BEGIN_USER_CODE
```

Comment:
It seems like LP_BEGIN_USER_CODE and LP_END_USER_CODE are in reverse order.

Reply:
No, it should be like that, because the method that is triggering `onStartResponse` has `LP_BEGIN_USER_CODE` at beginning and `LP_END_USER_CODE` at the end. Look for method `triggerStartResponse` in Leanplum.m file